README.md

```diff
@@ -240,11 +240,13 @@ Whenever you create a swing, it includes a `codeswing.json` file, which defines
 - `template` - Indicates that this swing is intended to be used as a [template for new swings](#user-templates), and therefore, will appear in the list when creating a new swing. Defaults to `false`.
 
 - `readmeBehavior` - Indicates how the swing's [readme](#readme) (if it has one) will be rendered to the end-user. Defaults to `previewHeader`.
--
+
 - `tutorial` - Indicates that this swing is intended to be used as a [multi-step tutorial](#tutorials). When set, this property indicates the title of the tutorial.
 
 - `input` - Indicates that this swing requires [user input](#swing-input), and also specifies an optional input file name, prompt message and completion message.
 
+- `themePreview` - Indicates that this swing is better viewed with Visual Studio Code theme applied to the webview. Note that this will take precedence over the user's configured default layout.
```

Comment:
Maybe call it "preview" instead of "webview"? I try to refer to the window as the preview, since that's a more user-facing term for it.